### ISSUE TITLE
SD results not same?

### ISSUE THREAD

**Issue by Aes-Sedai** [TheLastBen/fast-stable-diffusion#2536]
what has happened to SD? I been using same models for while now and I use same prompts I've been using and I get totally different results than before....I want to say I cannot make any NSFW images...is safety checker now enabled or something? 

also: what happened to the "PASTE" or "SAVE" button under prompt? delete/refresh are still there

**Comment by TheLastBen** [comment]
check in the A1111 repo for answers https://github.com/AUTOMATIC1111/stable-diffusion-webui

**Comment by Azura-13** [comment]
Hi, I'm noticing big issues in the render quality, with old prompts, even model renders look different, getting tattoos on skin when there should not be and would not be a few days ago. It feels like lora are fire when not on prompt.

Sorry no idea what the repo is or where it is, from your link ^^

**Comment by mdlieber99** [comment]
Yeah, I'd love to see more on this.  I couldn't find anything in the A1111 repo about this.  Do they say anything like they are going to improve this?  I know things will move towards SDXL, but there is a whole SD1.5 ecosystem of ckpts and Loras that produce great results that we should still be able to get.  I've completely stopped making anything online now (colab) until there is some clarity on this.

**Comment by xxxxxDantexxxxx** [comment]
same problem.

**Comment by mdlieber99** [comment]
Does anyone have any info on whether the developers (i.e. not Ben) are aware that they have messed up SD1.5 stuff and are planning to fix it in an update?

**Comment by godpunisher** [comment]
> check in the A1111 repo for answers https://github.com/AUTOMATIC1111/stable-diffusion-webui

Hi Ben, I'm also facing the same issue with SD 1.5. The generated images on google colab are different from local.
Also, the prompts became unruly. I used same checkpoint, sampler, hires fix, resolution, seed, sd web UI version and everything. But no luck.

I think, the main culprit is the python dependency v3.10.12 on colab. As per the A1111 recommendation, I use v3.10.6 on my local machine. I have shared the screenshots of both the colab & local, please have a look at it. Thank you very much for your efforts. You are the Robbin hood for thousands of potato PC users like me. Please fix it.

My command-line arguments are _--xformers --no-half --autolaunch_



Google Colab A1111
![Google Colab A1111](https://github.com/TheLastBen/fast-stable-diffusion/assets/52278351/c3b5a267-d497-4ef4-bdb6-10f40bb230ec)


Local A1111
![Local A1111](https://github.com/TheLastBen/fast-stable-diffusion/assets/52278351/ad659b95-b5eb-4858-adc7-a65b4ba989ff)

**Comment by mdlieber99** [comment]
I'm the original poster.  I figured out my problem thanks to this checklist: https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/13093
In my case, I use a bunch of embeddings in my negative prompts and I hadn't moved a copy to the new SD embeddings folder when I updated A1111.  Now that I have, my old prompts are reproduced about 99% the same.  Hope that checklist is helpful to others too.
